### PR TITLE
implement react-router-dom, basic app structutre setup

### PR DESCRIPTION
**Pull request by adampaulsackfield**

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { Link, NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+	return (
+		<div>
+			<Link to='/'>Navbar</Link>
```
Comment (jadelandeg) [inline src/components/Navbar.jsx:8]: Does this need to be a link?

**Review by jadelandeg — CHANGES_REQUESTED** [review]
Looking good! Not sure if nav bar itself needs to be a link though?

**Comment by adampaulsackfield** [comment]
Hi, yes it does need to be a link. It will be the logo and will link back to home. I just used the worst placeholder name going.

It can’t be a navlink aa they’re used to have active classes I believe.

**Review by jadelandeg — APPROVED** [review]
Approving!

**Pull request merged**